### ISSUE TITLE
Possible accessibility application

### ISSUE THREAD

**Issue by Piciok** [erlep/Konica-Minolta#1]
Hello,

I have just come across your repository as I was searching for possible solution to my problem. It seems that your work might be close to what I would like to achieve if I understand it correctly.

My usecase:
I am a blind employee at a workplace where one of my tasks involves operations on documents I have to scan using a KonnicaMinolta Bizhub 458 multifunctional device. The device is equipped with a touch screen that I cannot use. The accessibility solutions provided by the manufacturer will sadly not work with us for various reasons so I thought there may be a workaround allowing me to operate the machine anyway.

So far, I have tried handling the situation by installing the Bizhub Remote Access app on a smartphone. It turned out however, that what the app does is present a graphical image of the touch screen and allows for carrying out tasks by tapping on the same areas that would otherwise be accessed physically. This will not work with me as this representation does not map any native controls which my screen reader can hook into to read content. To find another solution, I have turned to Github which is how I came across your script.

Do you think your script could allow me to carry out the scanning of a document independently throughout the entire operation without having to access the physical touch screen of the device? Would any modifications be required in order to achieve that? Would you be willing to help out with adapting the script to do that? Sadly, I am not a programmer myself but I could setup a contact with somebody at our IT department if required.

I would be grateful for any feedback.

Have a nice day.

**Comment by erlep** [comment]
Hello, 
my python script automates the task related to downloading of already scanned document. 
Your user case is to instruct Bizhub to scan document. I have some quick investigation for you, but I haven't find any solution.

The device is equipped not only with a touch screen, but there are also buttons. Maybe it would be possible to setup these buttons to do some predefined scanning. I don't know how. Perhaps it would know in Konica Minolta service.
Best Regards & Have a nice day too.
